Kort om pre-spec-paper (#6)
SpecialeBajs/speciale-rapport: article/Introduction/pre-speciale-report.tex

\subsection{Pre specialization paper}

> In preparation for our master thesis we spend a semester gaining knowledge in the field of recommendation systems.

Reviewer: ```suggestion
In preparation for our master thesis, we spend a semester gaining knowledge in the field of recommendation systems.
```

> In preparation for our master thesis, we spend a semester gaining knowledge in the field of recommendation systems.

Reviewer 2: ```suggestion
In preparation for our master thesis, we spent a semester gaining knowledge in the field of recommendation systems.
```

We wrote a paper called \textit{Exploring Recommender Systems with Graph Convolutional Networks}.

> In this paper we studied recommendation systems using GCN's to achieve state of the art results.

Reviewer: ```suggestion
In this paper, we studied recommendation systems using GCN's to achieve state of the art results.
```

> In this paper, we studied recommendation systems using GCN's to achieve state of the art results.

Reviewer 2: ```suggestion
In this paper, we studied recommendation systems using Graph Convolutional Networks (GCN) to achieve state of the art results.
```

> In particular we looked at the models NGCF, PUP and LightGCN.

Reviewer: ```suggestion
In particular, we looked at the models NGCF, PUP and LightGCN.
```

> In particular, we looked at the models NGCF, PUP and LightGCN.

Reviewer 2: ```suggestion
In particular, we looked at the models Neural Graph Collaborative Filtering (NGCF), Price-aware User Preferencemodeling (PUP) and LightGCN.
```

> In particular, we looked at the models Neural Graph Collaborative Filtering (NGCF), Price-aware User Preferencemodeling (PUP) and LightGCN.

Reviewer: ```suggestion
In particular, we looked at the models Neural Graph Collaborative Filtering (NGCF), Price-aware User Preference-modeling (PUP) and LightGCN.
```

> PUP and LightGCN is both inspired by NGCF, where LightGCN removes complexity operations from NGCF and PUP adds side information in form of prices and categories.

Reviewer: ```suggestion
PUP and LightGCN are both inspired by NGCF, where LightGCN removes complexity operations from NGCF and PUP adds side information in form of prices and categories.
```

> Both achieved better performance in NDCG compared to NGCF.

Reviewer: ```suggestion
Both achieved better performance in NDCG compared to NGCF.
We were however not able to achieve the results, where PUP performed better than NGCF in our own experiments.
```

> Both achieved better performance in NDCG compared to NGCF.

Reviewer 2: ```suggestion
Both achieved better performance in NDCG and recall compared to NGCF.
```

> Because of this we thought it would be interesting to extend LightGCN with price and categories to see if it would improve performance.

Reviewer: ```suggestion
Because of this, we thought it would be interesting to extend LightGCN with price and categories to see if it would improve performance.
```
